doc/api/v0.yaml

```diff
@@ -2259,36 +2272,130 @@ paths:
+                                                description: upper bound of the available options, maximum being 16 (not included)
+                                                minimum: 1
+                                                maximum: 15
+                              Private:
+                                type: object
+                                description: Private tally result
+                                required: false
+                                properties:
+                                  state:
+                                    type: object
+                                    properties:
+                                      Encrypted:
+                                        type: object
+                                        required: false
+                                        properties:
+                                          encrypted_tally:
+                                            type: string
+                                            description: Base64-encoded encrypted tally bytes
+                                          total_stake:
+                                            type: integer
+                                      Decrypted:
+                                        type: object
+                                        required: false
+                                        properties:
+                                          result:
+                                            type: object
+                                            properties:
+                                              results:
+                                                type: array
+                                                items:
+                                                  type: integer
+                                              options:
+                                                type: object
+                                                properties:
+                                                  range:
+                                                    type: object
+                                                    properties:
+                                                      start:
+                                                        type: integer
+                                                        description: starting value of the vote option, starting from 0
+                                                        minimum: 0
+                                                        maximum: 15
+                                                      end:
+                                                        type: integer
+                                                        description: upper bound of the available options, maximum being 16 (not included)
+                                                        minimum: 1
+                                                        maximum: 15
```

Comment:
this could be abstracted to avoid repetition between public and private tally

Reply:
Updated